CHANGELOG.md

```diff
@@ -4,6 +4,35 @@
 
 💥 Breaking changes:
 
+- Make radios and checkboxes components easier to link to from the error summary component
+
+  The first input in the radios and checkboxes will have an `id` and `name` attribute that match, allowing for easier generation of error summary links.
+
+  You should check that clicking an error link for a radios or checkboxes results in the first input being focused.
```

Comment:
Is it clear enough what we mean by 'error link'? Would it be better to use something like:

```suggestion
  You should check that clicking a link from the error summary to radios or checkboxes results in the first input being focused.
```